src/main/java/delete_event_use_case/DeleteEventDataAccess.java

```diff
@@ -0,0 +1,33 @@
+package delete_event_use_case;
+
+import data_access.DataAccessDay;
+import data_access.DataAccessEvent;
+import data_access.WeekDataAccess;
+
+import java.util.HashMap;
+import java.util.ArrayList;
+
+/**
+ * Delete Event Data Access class.
+ * @author : Khizer Ahmad
+ * @created: November 19th 2022
+ * @since : 2022-11-21, Mon.
+ **/
+public class DeleteEventDataAccess extends WeekDataAccess implements DeleteEventDsGateway {
+
+    @Override
+    public boolean eventExists(int dayIndex, String eventTitle) {
+        DataAccessDay day = super.getDays().get(dayIndex);
+        return day.getEvents().containsKey(eventTitle);
+        //this should always return true if event is stored correctly
```

Comment:
I think expecting it to be stored properly is okay however we should keep a fail view in case or something goes wrong, I feel delete may be the most error-prone simply by its task which is why we should have a fail view.